appium/webdriver/extensions/flutter/flutter_commands.py

```diff
@@ -0,0 +1,166 @@
+            element: The element to double-click on. This parameter is required.
+            offset: The x and y offsets from the element to click at. If not specified, the click is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('doubleClick', opts)
+    
+    def perform_long_press(self, element: WebElement, offset: Optional[Tuple[int, int]] = None) -> None:
+        """
+        Performs a long press on the given element, with an optional offset.
+
+        Args:
+            element: The element to perform the long press on. This parameter is required.
+            offset: The x and y offsets from the element to perform the long press at. If not specified, the long press is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('longPress', opts)
+            
+    def perform_drag_and_drop(self, source: WebElement, target: WebElement) -> None:
+        """
+        Performs a drag-and-drop operation from a source element to a target element.
+
+        Args:
+            source: The element to drag from.
+            target: The element to drop onto.
+            
+        Returns:
+            None:
+        """
+        self.execute_flutter_command('dragAndDrop', {'source': source, 'target': target})
+        
+    def scroll_till_visible(self, scroll_to: FlutterFinder, scroll_direction: Optional[str] = 'down', **opts: Any) -> WebElement:
+        """
+        Scrolls until the specified element becomes visible.
+
+        Args:
+            scroll_to: The Flutter element to scroll to.
+            scroll_direction: The direction to scroll up/down. Defaults to 'down'.
+                
+        KeywordArgs:
+                scrollView (str): The view of the scroll.
+                delta (int): delta for the scroll
+                maxScrolls (int): Max times to scroll
+                settleBetweenScrollsTimeout (int): settle timeout
+                dragDuration (int): time gap between each scroll
+        
+        Returns:
+            Webelement: scrolled element
+        """
+        opts['finder'] = scroll_to.to_dict()
+        opts['scrollDirection'] = scroll_direction
+        return self.execute_flutter_command('scrollTillVisible', opts)
+        
+    def inject_mock_image(self, value: str) -> str:
+        """
+        Injects a mock image to the device. The input can be a file path or a base64-encoded string.
+
+        Args:
+            value: The file path of the image or a base64-encoded string.
+            
+        Returns:
+            str: Image ID of the injected image.
+        """
+        import os
```

Comment:
I think this import can be on the top of this file instead of in method.

Reply:
Thanks! Fixed